modules/network/src/main/java/org/apache/ignite/internal/network/netty/OutboundEncoder.java

```diff
@@ -41,6 +41,8 @@ public class OutboundEncoder extends MessageToMessageEncoder<OutNetworkObject> {
     /** Handler name. */
     public static final String NAME = "outbound-encoder";
 
+    private static final int IO_BUFFER_CAPACITY = 16 * 1024;
```

Comment:
I assume, this number is an optimal size? Or just a guess?

Reply:
A ran a few benchmarks. 16kb buffer allows to reach the same network utilization for big messages as 100kb buffer, also in benchmarks using our KV/SQL APIs it behaves on par (giving the same improvement for the scenario of PUTs containing 10k entries each). On the other hand, 16kb is substantially smaller than 100kb.

So I cannot claim that 16kb is an optimal value, but it works 'pretty well'. We can change it in the future if we realize that it's not optimal (this may require a lot of benchmarking).